hooks/post_gen_project.py

```diff
@@ -21,18 +21,18 @@
 msg = dedent(
     """
     %(separator)s
-    Tutorials:     http://docs.pylonsproject.org/projects/pyramid_tutorials/en/latest/
     Documentation: http://docs.pylonsproject.org/projects/pyramid/en/latest/
+    Tutorials:     http://docs.pylonsproject.org/projects/pyramid_tutorials/en/latest/
     Twitter:       https://twitter.com/trypyramid
     Mailing List:  https://groups.google.com/forum/#!forum/pylons-discuss
     Welcome to Pyramid.  Sorry for the convenience.
     %(separator)s
 
-    Create a virtualenv for the project:
+    Create a virtual environment for the project:
         cd {{ cookiecutter.repo_name }}
         %(venv_cmd)s %(venv)s
 
-    Install the project into the virtualenv:
+    Install the project into the virtual environment:
         %(venv)s/bin/pip install -e .[testing]
```

Comment:
You want to do:
     
        %(venv)s/bin/pip install -e ".[testing]"

Because some shells (zsh) treat [ and ] as a special characters.